apps/src/javalab/redux/consoleRedux.ts

```diff
@@ -0,0 +1,61 @@
+// Redux store for state that relates to the Java Lab console.
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+
+interface ConsoleLog {
```

Comment:
Just curious - will we be using `interface` instead of `type` as our standard approach for state? Just reading documentation about how we can declare either.

Reply:
Yep! The [typescript docs](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces) are mostly un-opinionated about which to use, but do say " If you would like a heuristic, use interface until you need to use features from type.", so we've been going with that so far.

Reply:
Thanks for the explanation! Looking forward to migrating to TS.